app/views/invoices/show.html.erb

```diff
@@ -0,0 +1,29 @@
+<h1> Invoice <%= @invoice.id %> </h1>
+
+<div class="invoice">
+  Invoice ID: <%= @invoice.id %> <br>
+  Invoice Status: <%= @invoice.status %> <br>
+  Invoice Created At: <%= @invoice.created_at.strftime("%A, %B %d, %Y") %> <br>
+  Invoice Customer: <%= @invoice.customer.full_name %>
+</div>
+
+<br>
+
+<div class="items">
+  <% @invoice.items.each do |item| %>
+    <div id="item-<%=item.id%>">
+      Item Name: <%= item.name %> <br>
+      Item Quantity: <%= @invoice.find_invoice_item(@invoice, item).first.quantity %> <br>
+      Item Price: $<%= number_with_precision ((item.unit_price.to_f) / 100), precision: 2, delimiter: "," %> <br>
+      Invoice Status: <%= @invoice.find_invoice_item(@invoice, item).first.status %> 
```

Comment:
same thing for this method string as well.

Reply:
The find_by was a great suggestion, I really appreciated that!